package.json

```diff
@@ -42,6 +42,7 @@
     "hapi-swagger": "^5.1.0",
     "inert": "^3.2.1",
     "screwdriver-plugin-builds": "^1.0.1",
+    "screwdriver-plugin-jobs": "^1.0.0",
     "screwdriver-plugin-pipelines": "0.0.2",
```

Comment:
Let's bump plugin-pipelines version so that we don't need to keep pinning it:
https://github.com/screwdriver-cd/plugin-pipelines/pull/4